lib/improver/utilities/filename.py

```diff
@@ -49,6 +54,9 @@ def generate_file_name(cube, parameter=None):
         parameter (str):
             Optional parameter name to use in the output filename rather than
             taking the name of the cube diagnostic.
+        include_period (bool):
+            Keyword argument to indicate whether a period, accumulation or
+            time window identifier should be included within the filename.
 
     Returns:
         filename (str):
```

Comment:
You need to update your Raises section too.

Reply:
Updated Raises section.